src/internal/packager/images/common.go

```diff
@@ -18,6 +19,7 @@ type ImgConfig struct {
 	Insecure bool
 }
 
+// New creates a new container image config with the provided image config.
 func New(config *ImgConfig) *ImgConfig {
```

Comment:
Just leaving a comment here that this is a really odd "constructor" (gimme the object for free and I'll sell it back to you!)

Reply:
(this is an unrelated comment - we could make this an issue, or just resolve this conversation and memory hole it forever)

Reply:
It's not used, maybe we just kill it now?

Reply:
agree it's weird, must have been the victim of merge conflict resolutions or refactors...

Reply:
It actually doesn't look to be used so yeah I agree to kill.  Gonna push an update to the branch.  I think I can also fix the period thing too.

Reply:
Removed this constructor